src/test/java/org/prebid/cache/handlers/PostCacheHandlerTests.java

```diff
@@ -74,4 +96,29 @@ void testVerifySave() {
                 .expectComplete()
                 .verify();
     }
+
+    @Test
+    void testExternalUUIDInvalid() {
+        //given
+        cacheConfig.setAllowExternalUUID(false);
```

Comment:
I wouldn't do this since Spring Test could cache context and beans between tests so it's better to create new config object just for this test